app/views/category/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class='navbar navbar-login'>
+  <%= stylesheet_link_tag "categories" %>
+  <%= link_to 'Back', splash_screen_path, class: 'back-link' %>
+  <h2 style="color: white; background-color: transparent;">CATEGORIES</h2>
+  <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "sign_out text_white" %>
+</div>
+
+<div class="categories">
+  <% if @categories.length == 0 %>
+    <p class="empty_data">No category yet. Add a new one!!!</p>
+  <% else %>
+    <% @categories.each do |category| %>
+      <%= link_to category_expense_index_path category do %>
+        <%= render category %>
+      <% end %>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="new_category_btn">
+  <%= link_to "Add Category", new_category_path, class: "fcc-btn" %>
```

Comment:
- [ ] As mentioned by the previous reviewer kindly follow the given design 🙃 

Your design

<img width="374" alt="Screenshot 2023-09-16 at 19 27 39" src="https://github.com/yosaddis/ruby-on-rails-capstone/assets/74501523/2b6f4302-8d44-45c2-89e8-aa724def0479">

Given design

<img width="371" alt="Screenshot 2023-09-16 at 19 33 42" src="https://github.com/yosaddis/ruby-on-rails-capstone/assets/74501523/80c9dee3-6948-4971-8131-cef207380961">